src/datachain/catalog/catalog.py

```diff
@@ -1805,14 +1772,15 @@ def apply_udf(
     def query(
         self,
         query_script: str,
-        envs: Optional[Mapping[str, str]] = None,
-        python_executable: Optional[str] = None,
+        env: Optional[Mapping[str, str]] = None,
+        python_executable: str = sys.executable,
```

Comment:
Changed `python_executable` to default to `sys.executable` and not take a `None` value.